assets/src/scripts/monitoring-location/components/hydrograph/parameters.js

```diff
@@ -17,109 +15,218 @@ import {getAvailableParameters} from './selectors/parameter-data';
+        .attr('type', 'radio')
+        .attr('name', 'parameter-selection')
+        .attr('value', `${parameter.parameterCode}`)
+        .call(link(store, (inputElem, selectedParameterCode) => {
+            inputElem.property('checked', parameter.parameterCode === selectedParameterCode ? true : null);
+        }, getSelectedParameterCode));
+    radioButtonDiv.append('label')
+        .attr('class', 'usa-radio__label')
+        .attr('for', `radio-${parameter.parameterCode}`);
+    gridRowInnerWithRadioButton.append('div')
+        .attr('class', 'grid-col-7 description__param-select')
+        .append('div')
+        .text(`${parameter.description}`);
+    gridRowInnerWithRadioButton.append('div')
+        .attr('class', 'grid-col-auto period-of-record__param-select')
+        .text(`${parameter.periodOfRecord.begin_date} to ${parameter.periodOfRecord.end_date}`);
+    const radioRowExpansionControlDiv = gridRowInnerWithRadioButton.append('div')
+        .attr('class', 'grid-col open-close-radio_button_row');
+    drawRowExpansionControl(radioRowExpansionControlDiv, parameter, 'desktop');
+};
+
+/*
+* Helper function that draws a row containing the controls for the WaterAlert subscription.
+* @param {String} siteno - A unique identifier for the monitoring location
+* @param {Object} Element - The target element to append the row
+* @param {Object} parameter - Contains details about the current parameter code
+*/
+const drawWaterAlertRow = function(siteno, element, parameter) {
+    const gridRowInnerWaterAlert = element.append('div')
+        .attr('class', 'grid-row grid-row-inner');
+
+        gridRowInnerWaterAlert.append('div')
+            .attr('id', `wateralert-row-${parameter.parameterCode}`)
+            .attr('class', 'grid-col grid-offset-1 wateralert-row')
+            .append('a')
+            .attr('href', `${config.WATERALERT_SUBSCRIPTION}/?site_no=${siteno}&parm=${parameter.parameterCode}`)
+            .attr('target', '_blank')
+            .attr('class', 'water-alert-cell usa-tooltip')
             .attr('data-position', 'left')
             .attr('data-classes', 'width-full tablet:width-auto')
-            .attr('title', d.waterAlert.tooltipText)
-            .text(d.waterAlert.displayText);
+            .attr('title', parameter.waterAlert.tooltipText)
+            .text(parameter.waterAlert.displayText);
+};
+
+/*
+* A main function that creates the parameter selection list
+* @param {Object} Store - The application Redux state
+* @param {Object} Element - The target element to append the selection list
+* @param {String} siteno - A unique identifier for the monitoring location
+* */
+export const drawSelectionList = function(container, store, siteno) {
+    const parameters = getAvailableParameters(store.getState());
+
+    if (!Object.keys(parameters).length) {
+        return;
+    }
+    // Add the primary parameter selection container.
+    container.append('h2')
```

Comment:
I don't think we should use h2. This has semantic meaning in the markup. Use the style sheet to get the text style/font/weight that we want.

Reply:
I was undecided on this, but thought 'h2' was acceptable since, in my mind, this is a section subheading. But I'm happy to go with a styled text element as that works just as well.

Reply:
We are using h2 (and h3 inconsistantly) on some of the accordion titles. I think this is essentially a contination of the main section which includes the hydrograph.